ceph_deploy/osd.py

```diff
@@ -593,7 +594,7 @@ def print_osd(logger, hostname, osd_path, json_blob, metadata, journal=None):
     logger.info('%-14s %s' % ('Reweight', json_blob.get('reweight')))
     if journal:
         logger.info('Journal: %s' % journal)
-    for k, v in metadata.items():
+    for k, v in list(metadata.items()):
```

Comment:
Yes, the automatically-added `list` is not needed. I will fix things like these in the future.